narwhals/translate.py

```diff
@@ -46,6 +45,15 @@
 
 T = TypeVar("T")
 
+NON_TEMPORAL_SCALAR_TYPES = (
```

Comment:
Kind of related, I am running a test, and getting a raise if None enters this function. Should we add it to non temporal scalar types?

Reply:
_very_ good point, thanks!